src/app/@core/utils/languages.service.ts

```diff
@@ -5,56 +5,98 @@ import { TranslateService } from '@ngx-translate/core';
 import { map, tap } from 'rxjs/operators';
 import { UserData } from '../interfaces/users';
 
+import { registerLocaleData } from '@angular/common';
+import fr from '@angular/common/locales/fr';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/HEPIA-Association/association-WebClient/pullRequest?prid=7914646)
```suggestion
import fr from "@angular/common/locales/fr";
```